pytools/tag.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
```

Comment:
This is not consistent with the Py3.6+ promise in `setup.py` (which I'd like to retain)

Reply:
Okay, I think this requires making the forward reference to DottedName a string then.